presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSplit.java

```diff
@@ -43,6 +43,7 @@
     private final int partitionId;
     private final long start;
     private final long end;
+    private final long timestamp;
```

Comment:
Hi, Thanks for raising this PR. Since we are computing the Timestamp value from the `messageAndOffset` why do we need to fetch/maintain it in the split ?

Reply:
Hi @Praveen2112, Thanks for reviewing it. You are right, here we don't need to maintain Timestamp with offset. We can directly only take it from messageAndOffset. I have made the changes required.